bin/index.js

```diff
@@ -1,4 +1,19 @@
 #!/usr/bin/env node
-const newProject = require('../index');
+/* eslint-env node */
+/* eslint-disable no-console */
+const chalk = require('chalk');
 
-newProject(process.argv[2]);
+const blue = chalk.blueBright;
+const cyan = chalk.cyan;
+
+console.log(
+  `🤗  Hey! ${blue('creuna-new')} has been deprecated. Use ${blue(
+    '@creuna/cli'
+  )} instead! It's much nicer.
+  • ${blue('yarn remove -g @creuna/create-react-app')} or ${cyan(
```

Comment:
Global in Yarn is done using `yarn global <command>` not with a `-g` flag.

Reply:
Fixed